dpytools/http/upload.py

```diff
@@ -0,0 +1,131 @@
+import datetime
+import os
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Tuple, Union
+
+from dpytools.http.base import BaseHttpClient
+
+
+class UploadClient(BaseHttpClient):
+    def __init__(self, upload_url: str):
+        # Inherit backoff_max value from BaseHTTPClient.__init__
+        super().__init__()
+        self.upload_url = upload_url
+        # For security purposes, `self.access_token` should be set using a FLORENCE_TOKEN environment variable
+        self.access_token = os.environ.get("FLORENCE_TOKEN")
```

Comment:
Not sure if this is for this ticket or what the solution is just adding my thoughts..
FLORENCE_TOKEN is generated by using a Florence login and password and it is only valid for a set time so you would need to generate a token and change it in your environment variables each time you wanted to run this, which can work but only if it running locally. 
This wouldn't work when it is running through AWS, it would need either a way of generating the token or would need a token that doesn't expire. I suspect this would be for the platform team to advise/resolve

Reply:
Nasty.

Sarah - can you just change upload method(s) to accept a florence token as an argument rather than look for an env var.

that way no rewrites are needed here and if need be we can write an auth client or similar to get a token.